packages/aws-cdk/README.md

```diff
@@ -746,6 +746,32 @@ NOTICES
 If you don’t want to see a notice anymore, use "cdk acknowledge <id>". For example, "cdk acknowledge 16603".
 ```
 
+### `cdk migrate`
+
+⚠️**CAUTION**⚠️ CDK migrate is currently experimental. We make no guarantees about the outcome or stability of the Command or it's output.
```

Comment:
Experimental means that it might have breaking changes in the future. Saying we make no guarantees about the commands output sounds like the command may or may not do what the documentation says, and that is not the case. 

Reusing the exact phrasing from another experimental command:
```suggestion
⚠️**CAUTION**⚠️ This command is considered experimental, and might have breaking changes in the future.
```

Reply:
This is what we currently output when a user runs our command. Should we change it in both places?

Reply:
I would be in favor of that, yes